WorkloadManagementSystem/Executor/JobScheduling.py

```diff
@@ -453,17 +453,15 @@ def __sendToTQ(self, jobState, jobManifest, sites, bannedSites, onlineSites=None
 
     # Job multivalue requirement keys are specified as singles in the job descriptions
     # but for backward compatibility can be also plurals
-    for key in ('SubmitPools', "SubmitPool", "GridMiddleware", "PilotTypes", "PilotType",
+    for key in ('SubmitPools', "SubmitPool",
                 "JobType", "GridRequiredCEs", "GridCE", "Tags"):
       reqKey = key
       if key == "JobType":
         reqKey = "JobTypes"
-      elif key == "GridRequiredCEs" or key == "GridCE":
+      elif key == "GridRequiredCEs" or key == "GridCE":  # Remove obsolete GridRequiredCEs
         reqKey = "GridCEs"
-      elif key == "SubmitPools" or key == "SubmitPool":
+      elif key == "SubmitPools" or key == "SubmitPool":  # Check: obsolete?
```

Comment:
If even this is not mandatory, it can be useful. The idea was to use it for pilots with some specific requirements to their Computing Elements. This was not yet realized, but can be done especially for the HPC CEs.

Reply:
I know, indeed I have not removed the option, but I can't figure out if the correct one would be "SubmitPools" or "SubmitPool". I clarify the comment.